libnethack/include/rtp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef RTP_H
+#define RTP_H
+
+#include "extern.h"
+#include "hack.h"
+#include "monst.h"
+
+#define RTP_NAME_COUNT 16
+
+// These are added in order of when I got permission
+#define RTP_NAMES { \
+    "Liam Middlebrook", \
+    "Stephanie Miller", \
+    "McSaucy", \
+    "Russ Harmon", \
+    "Jordan Rodgers", \
+    "Ethan House", \
+    "Rob Glossop", \
+    "Grant Cohoe", \
+    "Steve Greene", \
+    "William Dignazio", \
+    "Chris Lockfort", \
+    "Will Orr", \
+    "Derek Gonyeo", \
+    "Kevin Thompson", \
+    "Angelo DiNardi", \
+    "Dan Willemsen", \
+}
+
+// This is a pretty crappy WAR, but for now it makes more sense than writing a
+// more elegant solution. Hopefully this will become a bigger problem in the
+// future
+
+// The expanded format for this should be something like ((i == 1) || (i == 17))
+#define FEMALE_RTPS(i) ((i == 1))
```

Comment:
Ew. Gross. Please don't hard-code the indexes of female RTPs, especially not using a macro.

This is easy to fix by declaring the array I mentioned above with a struct. E.g.

```c
#include <stdbool.h>
...
const struct {
  const char * const name;
  const bool female;
} rtps[] = {
  { "Liam Middlebrook", /*female=*/false },
  { "Stephanie Miller", /*female=*/true },
  ...
};
```